frontend/src/statics/Statics.ts

```diff
@@ -0,0 +1,65 @@
+export const disciplines:string[] = [
```

Comment:
Is this supposed to be an enum and/ or are those selectable options?
Clarify please and improve the naming of this file, because I couldn't tell it's purpose :warning: